solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -445,6 +463,79 @@ contract WalletRegistryGovernance is Ownable {
         newMaliciousDkgResultNotificationRewardMultiplier = 0;
     }
 
+    /// @notice Begins the dkg result submission gas update process.
+    /// @dev Can be called only by the contract owner.
+    /// @param _newDkgResultSubmissionGas New DKG result submission gas.
+    function beginDkgResultSubmissionGasUpdate(
+        uint256 _newDkgResultSubmissionGas
+    ) external onlyOwner {
+        /* solhint-disable not-rely-on-time */
+        require(
+            _newDkgResultSubmissionGas != 0,
+            "DKG result submission gas cannot be zero"
+        );
+
+        newDkgResultSubmissionGas = _newDkgResultSubmissionGas;
+        dkgResultSubmissionGasChangeInitiated = block.timestamp;
+        emit DkgResultSubmissionGasUpdateStarted(
+            newDkgResultSubmissionGas,
```

Comment:
I think that if we use `_newDkgResultSubmissionGas` we could get a lower gas consumption. We use this trick in other functions. Same comment to `beginDkgApprovalGasOffsetUpdate`.

Reply:
[1e3d178](https://github.com/keep-network/keep-core/pull/2856/commits/1e3d178e150721546d590065511c40fad294e98b)